From cafb265fb9b3ca6ceeed6e3a57edb2eb5def409e Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 20 Oct 2017 16:45:53 -0400
Subject: [PATCH 4641/5855] drm/amdgpu: Remove immutable flag from
 freesync_capable property

Atomic drivers should not use legacy properties but atomic properties
don't support the immutable flag. Remove the immutable flag for now.

This will be followed by a bunch of DC patches to start treating
freesync_capable as an atomic property on a connector_state.

Eventually we'll want to remove the these properties and replace them
with new atomic adaptive sync properties that we define in DRM.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Andrey Grodzovsky <andrey.grodzovsky@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_display.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_display.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_display.c
index 2170f17..8667ffb 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_display.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_display.c
@@ -658,7 +658,7 @@ int amdgpu_modeset_create_props(struct amdgpu_device *adev)
 			return -ENOMEM;
 		adev->mode_info.freesync_capable_property =
 			drm_property_create_bool(adev->ddev,
-						 DRM_MODE_PROP_IMMUTABLE,
+						 0,
 						 "freesync_capable");
 		if (!adev->mode_info.freesync_capable_property)
 			return -ENOMEM;
-- 
2.7.4

